lib/libcouchbase/query_n1ql.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true, encoding: ASCII-8BIT
+
+module Libcouchbase
+    class QueryN1QL
+        N1P_QUERY_STATEMENT = 1
+        N1P_CONSISTENCY_REQUEST = 2
```

Comment:
why do you overload all class instead of `class_eval`?

Reply:
query n1ql crash with `class_eval`